presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -70,6 +70,7 @@
     WARNING_AS_ERROR(0x0000_002C, USER_ERROR),
     INVALID_ARGUMENTS(0x0000_002D, USER_ERROR),
     EXCEEDED_PLAN_NODE_LIMIT(0x0000_002E, USER_ERROR),
+    MISSING_TABLE(0x0000_002F, USER_ERROR),
```

Comment:
Why not reuse the one in `SemanticErrorCode`?

Reply:
The node classes are used to format the message.  I will leave this clean up for later PR. Let me know if you feel strongly about it.

Reply:
I think it's better to have one `MISSING_TABLE` in one place. It's ok to clean it up either in this or next PR.